src/Workspaces/VisualBasicTest/VisualBasicExtensionsTests.vb

```diff
@@ -0,0 +1,37 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Text
+Imports Microsoft.CodeAnalysis.Text
+Imports Roslyn.Test.Utilities
+Imports Xunit
+Imports Microsoft.CodeAnalysis.VisualBasic.Extensions.SyntaxTreeExtensions
+Imports System.Threading
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
+
+    Public Class VisualBasicExtensionsTests
+        <WorkItem(6536, "https://github.com/dotnet/roslyn/issues/6536")>
+        <Fact>
+        Public Sub TestFindTrivia_NoStackOverflowOnLargeExpression()
```

Comment:
The test should be moved to the compiler's tests. For example into src\Compilers\VisualBasic\Test\Syntax\Syntax\SyntaxTriviaListTests.vb